styla.php

```diff
@@ -252,9 +252,9 @@ private function buildContentKey() {
             $contentKey = isset($_REQUEST["origUrl"]) ? $_REQUEST["origUrl"] : "/";
 
             if($this->rootPath !== "/"){
```

Comment:
2. second issue was that as the rootPath was being passed as NULL, this condition evaluated to true, so it was defining `$path` as `/`, deleting all the slashes on the url, so the SEO API was returning a 404 in all cases except for root url.